static/scripts/helpers/AlertMessageController.js

```diff
@@ -1,14 +1,14 @@
+/* eslint-disable max-len */
```

Comment:
Can we just decrease the lengths of the needed lines? If not, we can disable the eslint rule just for spesific lines